dist/recline.js

```diff
@@ -3227,13 +3240,7 @@ my.SlickGrid = Backbone.View.extend({
     }
 
     function sanitizeFieldName(name) {
-      var sanitized;
-      try{
-        sanitized = $(name).text();
-      } catch(e){
-        sanitized = '';
-      }
-      return (name !== sanitized && sanitized !== '') ? sanitized : name;
+      return $('<div>').text(name).html();
```

Comment:
![screen shot 2017-01-06 at 1 49 06 pm](https://cloud.githubusercontent.com/assets/428070/21725374/f10311e0-d416-11e6-8cd8-3e0fbadee43a.png)

apparently there's something in `okfn/master` that didn't got build and commited.